WebDriverAgentLib/Categories/XCUIElement+FBTap.m

```diff
@@ -23,7 +23,8 @@ - (BOOL)fb_tapWithError:(NSError **)error
   [self fb_waitUntilFrameIsStable];
   __block BOOL didSucceed;
   [FBRunLoopSpinner spinUntilCompletion:^(void(^completion)()){
-    [[XCEventGenerator sharedGenerator] tapAtPoint:self.lastSnapshot.hitPoint orientation:self.interfaceOrientation handler:^(XCSynthesizedEventRecord *record, NSError *commandError) {
+    CGPoint hitPoint = self.isHittable ? self.lastSnapshot.hitPoint : [self coordinateWithNormalizedOffset:CGVectorMake(0, 0)].screenPoint;
```

Comment:
correct this will tap top left corner. I think it doesnt matter since its a coordinate off the element itself. Its arbitrary where you tap on the element unless the elements behavior depends on where its tapped.